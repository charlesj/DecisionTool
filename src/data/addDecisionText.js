import { paths } from './paths'

export const addDecisionText = (currState, decisionText) => {
  return currState.setIn(paths.decisionText, decisionText)
}
