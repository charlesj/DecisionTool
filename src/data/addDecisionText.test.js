import { fromJS } from 'immutable'
import { addDecisionText } from './addDecisionText'

it('returns a new state map with the decision text in it', () => {
  const currentState = fromJS({})
  const decisionText = 'here is my new decision text'
  expect(addDecisionText(currentState, decisionText)).toMatchSnapshot()
})
