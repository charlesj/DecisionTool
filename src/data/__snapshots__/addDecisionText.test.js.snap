// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`returns a new state map with the decision text in it 1`] = `
Immutable.Map {
  "decisionText": "here is my new decision text",
}
`;
